packages/minecraft.rb

```diff
@@ -0,0 +1,61 @@
+require 'package'
+
+class Minecraft < Package
+  description 'A sandbox video game developed by Mojang Studios AB.'
+  homepage 'https://minecraft.net'
+  version '**filler**' # Not sure what to put here since its the launcher version not the game version.
```

Comment:
2.2.1867?  Run `alien -tc Minecraft.deb` to find the version.

Reply:
2.2.1867

Reply:
alien -tc wasn't working so i did
```shell
ar x Minecraft.deb
tar xf control.tar.gz
cat control
```

Reply:
The resulting file after `alien -tc ...` will have the version in the filename.

Reply:
ohhh i gotcha.